crypto/signature.go

```diff
@@ -23,6 +23,13 @@ type StablyMarshallable interface {
 //	_ = Signature(refs.Signature{}) // not recommended
 type Signature refs.Signature
 
+// NewSignature is a Signature instance constructor.
+func NewSignature(scheme Scheme, publicKey PublicKey, value []byte) Signature {
+	var s Signature
+	s.setFields(scheme, publicKey, value)
```

Comment:
why not public setters? i like constructors and have nothing against them but i would ask why the other structs in the SDK are not that convenient then

Reply:
We can't fix all of them now.

Reply:
sure, just want SDK to be consistent. won't stop me from `Approve` button

Reply:
we have #483. Well, nothing prevents me to do this in all packages

Reply:
yeap, that is the questions: why is the only one done, why now and what exactly does it solve? but the change itself is ok to me